WordPress/src/main/res/layout/quick_actions_block.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/quick_actions_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingBottom="@dimen/margin_my_site_quick_actions_bottom"
+    android:paddingEnd="@dimen/content_margin_site_row_start"
+    android:paddingStart="@dimen/content_margin_site_row_start"
+    android:paddingTop="@dimen/margin_extra_small"
+    android:clipChildren="false"
+    android:clipToPadding="false"
+    android:orientation="horizontal">
+
+    <RelativeLayout style="@style/MySiteQuickActionButtonContainer">
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/quick_action_stats_button"
+            style="@style/MySiteQuickActionButton"
+            android:src="@drawable/ic_stats_alt_white_24dp" />
+
+        <org.wordpress.android.widgets.WPTextView
+            style="@style/MySiteQuickActionButtonLabel"
+            android:layout_below="@+id/quick_action_stats_button"
+            android:text="@string/stats" />
+    </RelativeLayout>
+
+    <Space
```

Comment:
TIL 👍

Reply:
Btw, this layout was just copied from the old fragment. Once we update the way we show the Quick Start indicators, I believe we can have a simpler structure here using `ConstraintLayout` or something like that.